cmd/fetchbranches/README.md

```diff
@@ -0,0 +1,42 @@
+# Fetch branches tool
+
+## Overview
+
+The `fetch branches` tool is used to perform a fetch of a branch 
+locally to test and verify the changes before merging on master. 
+The `fetch branches` tool is designed to run in Semaphore and it 
+will only accept GitHub pull requests.
+The `fetch branches` tool uses Semaphore's environment variables 
+(PULL_REQUEST_NUMBER and SEMAPHORE_REPO_SLUG) to retrieve 
+information like the pull request number and the name of the owner 
+and the repository.
+
+## Validation Tags
+
+When raising a pull request, the commit message should be sign-off. 
```

Comment:
'should be sign-off' might be better as 'should be signed off', or maybe better still 'must have a Signed-off-by entry' or similar.